code/modules/shuttle/emergency.dm

```diff
@@ -411,21 +417,17 @@
 	return has_people && ((hijacker_count == 1) || (hijacker_count && !solo_hijack))
 
 /obj/docking_port/mobile/emergency/proc/is_hijacked_by_xenos()
-	var/has_xenos = FALSE
+	var/queen_alive_on_shuttle = FALSE
+	//checking all players
 	for(var/mob/living/player in GLOB.alive_mob_list)
-		if(issilicon(player)) //Borgs are technically dead anyways
+		if(issilicon(player)) //Borged queen brain is dead, skip
```

Comment:
why does it even bother skipping these conditions when it's checking istype anyway

Reply:
Because we don't want the game to fire the xeno event as a result of the queen being borged?

What are you asking me to change here?

Reply:
isalienqueen is not a mob type check? just check if the queen is an alien queen

Reply:
Okay yeah I can see how the redundancy isn't necessary here 👍